UI/src/app/dashboard/controllers/admin.js

```diff
@@ -326,5 +326,81 @@
                 }
             );
         }
+ 
+//Configuration settings functionality starts here
+        dashboardData.getGeneralConfig().then(processGeneralConfigResponse);
+        function processGeneralConfigResponse(data){
+            console.log(data)
+            if(data){
+                $scope.choices = data;
+            }else{
```

Comment:
What are these static ids ?

Reply:
For testing purpose we have added those lines, Now it has been removed